### ISSUE TITLE
Device Tracker not Working correctly on Asus Merlin with Mesh network

### ISSUE THREAD

**Issue by SoulSlayerPT** [ollo69/ha_asuswrt_custom#6]
Hi,

I am trying to use this custom integration because the officional one crasches my AX88U router with the SSH login issue. I already tried with HTTPS and HTTP and my phones never change from home to not_home status when i disable the wifi. Tried to change the time setting from 180seconds to another value and the phones are always in home status.

Am i doing something wrong in the configurations? Do i need to change any setting in the router?

My system is 1 AX88U main router(asus merlin firmware) and 3 XT8 units.

Can i provide any log to help me out?

Thank you

**Comment by ollo69** [comment]
Did you remove the native integration before configure this one? Can you download integration diagnostic and attach it here?

**Comment by SoulSlayerPT** [comment]
I have removed the native integration, yes.
Sure. I have removed the other tracked devices and only have my phone and the 3 XT8. Even if i disable the phone wifi the "last_activity" still updates the time.

[config_entry-asuswrt_custom-09b7328cb26871a38473af3375b6db61.json.txt](https://github.com/ollo69/ha_asuswrt_custom/files/10203049/config_entry-asuswrt_custom-09b7328cb26871a38473af3375b6db61.json.txt)
